conf/machine/nanopi-m1-plus.conf

```diff
@@ -0,0 +1,15 @@
+#@TYPE: Machine
+#@NAME: nanopi M1 Plus
+#@DESCRIPTION: Machine configuration for the FriendlyARM NanoPi Neo, based on the Allwinner H3 CPU
+
+require conf/machine/include/sun8i.inc
+
+PREFERRED_VERSION_u-boot = "v2017.11%"
+PREFERRED_VERSION_linux-mainline = "4.16+git%"
+
+KERNEL_DEVICETREE = "sun8i-h3-nanopi-m1-plus.dtb"
+UBOOT_MACHINE = "nanopi_m1_plus_defconfig"
+
+DEFAULTTUNE = "cortexa7hf-neon-vfpv4"
+
+MACHINE_FEATURES = "alsa apm keyboard rtc serial screen usbgadget usbhost vfat bluetooth wifi"
```

Comment:
Can you pls drop this line? Can be added in local.conf to not limit other users. Thanks.